tests/meta/demultiplex/test_housekeeper_storage_functions.py

```diff
@@ -227,3 +231,93 @@ def test_store_fastq_path_in_housekeeper_correct_tags(
     file: File = populated_housekeeper_api.get_files(bundle=sample_id).first()
     expected_tags: set[str] = {SequencingFileTag.FASTQ.value, novaseq6000_flow_cell.id, sample_id}
     assert {tag.name for tag in file.tags} == expected_tags
+
+
+@pytest.mark.parametrize(
+    "file_tag",
+    [SequencingFileTag.FASTQ, SequencingFileTag.SPRING, SequencingFileTag.SPRING_METADATA],
```

Comment:
Is parametrisation the correct approach here? I would say you need to check all three types of files every time you call the delete command. I would do a for loop over the file types with an assert statement inside in the THEN section.

Reply:
It's a valid approach but would take a lot more code. Keeping this